docpages/building/linux.md

```diff
@@ -37,27 +37,31 @@ Once installed to the `/usr/local` directory, you can make use of the library in
 g++ -std=c++17 mydppbot.cpp -o dppbot -ldpp
 ```
 
-If you are on **Arch Linux**, you will need to add `/usr/local` to your environment variables:
+If you are on **Arch Linux**:
 
+- You need to give proper permission to `libdpp.so` and `libdpp.so.10.0.29`
 ```bash
-export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
-export CPLUS_INCLUDE_PATH=/usr/local/include:$CPLUS_INCLUDE_PATH
+sudo chmod 644 /usr/local/lib/libdpp.so && sudo chmod +x /usr/local/lib/dpp.so
+sudo chmod 644 /usr/local/lib/libdpp.so.10.0.29 && sudo chmod +x /usr/local/lib/dpp.so.10.0.29
```

Comment:
Have you tried without chmod?

Reply:
> Have you tried without chmod?

Yes i did `ldd /usr/local/libdpp.so` and the output was `ldd: warning: you do not have execution permission for /usr/local/lib/libdpp.so.10.0.29'`  so yaa chmod is required

Reply:
Weird, did you `sudo make install`?

Reply:
> Weird, did you `sudo make install`?

yes

Reply:
Alrighty.

Reply:
are you sure you don't have a non standard umask?

Reply:
this will need to be updated every time we make a release. this is a pain point. should set your umask correct before make install

Reply:
> are you sure you don't have a non standard umask?

I have standard umask
```bash
ishank@Beast ~> umask
0022
```
standard umask is 0022 and this is what i have

Reply:
> this will need to be updated every time we make a release. this is a pain point. should set your umask correct before make install

we are building from source which already takes a lot of command, and can not be updated automaticaly even if you release new versions, we need to build new releases from source and then adding these small like to set permission is not a long process and also this is the only solution (as far as i have researched for `/usr/local/`)